docs/examples/cloud_trace_exporter/README.rst

```diff
@@ -20,15 +20,28 @@ To use this exporter you first need to:
     pip install opentelemetry-sdk
     pip install opentelemetry-exporter-cloud-trace
 
-* Run example
+* Run example locally
 
 .. code-block:: sh
 
+    cd opentelemetry-python/docs/examples/cloud_trace_exporter
     python basic_trace.py
 
 Checking Output
 --------------------------
 
 After running any of these examples, you can go to `Cloud Trace overview <https://console.cloud.google.com/traces/list>`_ to see the results.
 
-* `More information about exporters in general <https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html#configure-exporters-to-emit-spans-elsewhere>`_
\ No newline at end of file
+
+Further Reading
+--------------------------
+
+* `More information about exporters in general <https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html#configure-exporters-to-emit-spans-elsewhere>`_
+
+Troubleshooting
+--------------------------
+
+Running basic_trace.py hangs:
```

Comment:
nitpick: put these as RST subheadings? This enables these questions to show up in the TOC of a sphinx application, assuming sufficient depth is requested.

Also it makes it a little easier to read, even in markup format.